browser/src/control/jsdialog/Util.Dropdown.js

```diff
@@ -168,6 +170,14 @@ JSDialog.OpenDropdown = function (id, popupParent, entries, innerCallback, popup
 	};
 	L.Map.THIS.fire('closepopups'); // close popups if a dropdown menu is opened
 	L.Map.THIS.fire('jsdialog', {data: json, callback: generateCallback(entries)});
+
+	var dropDialog = L.Map.THIS.jsdialog.getDialog(dropdownWindowId);
```

Comment:
Please do not use `map.jsdialog` in the widget implementation.
Widget can be created by different containers which will fail here.